### PR TITLE
[GSoD 2021] Rendered openapi/CHANGELOG.md file in the History section of OpenAPI docs

### PR DESCRIPTION
**Pull request by iamrajiv**
Related: #1012

Screenshot 2021-06-05 at 5 57 23 PM

### CODE REVIEW

**Comment by iamrajiv** [comment]
Hi, @huan @Rohitesh-Kumar-Jain @proudofsimin I have made a PR [#11](https://github.com/wechaty/openapi/pull/11) in [wechaty/openapi](https://github.com/wechaty/openapi) repository which will add `CHANGELOG.md` file. So, as of now, the History section output will not load because PR [#11](https://github.com/wechaty/openapi/pull/11) is not yet merged if it gets merged then the section will load fine.

**Comment by proudofsimin** [comment]
Hi @iamrajiv,
Thank you for taking care of the version update history. Do you think if it's better to change the layout to this:

## History

This table shows a history of recent updates to the _openAPI_.

| Date | Change|
|------|--------|
|...       |...          |

**Comment by iamrajiv** [comment]
Hi, @proudofsimin. Unfortunately, we can't change the layout of the History section like that because it is rendered in real-time from [wechaty/openapi](https://github.com/wechaty/openapi). The layout will always look like this https://github.com/wechaty/wechaty/blob/master/CHANGELOG.md.

**Comment by proudofsimin** [comment]
Hi @iamrajiv, 
Then please remove me from the reviewer if the document structure is unable to change. Thank you.

**Review by Rohitesh-Kumar-Jain — DISMISSED** [review]
I am not the best person to review this, but it looks fine to me : )

**Review by huan — APPROVED** [review]
LGTM.

The `zero-md` looks great, this module gives me a reason to put `CHANGELOG.md` in our repos!

Please read and follow #1257, and feel free to help yourself to merge it by following the merge workflow.

Thank you very much!

File: .gitignore [hunk 1]
```diff
@@ -12,3 +12,4 @@ new-gh-pages/
 tmp/
 
 hash.file
+jekyll/_posts/2020-12-31-osschat-a-bridge-to-the-apache-way-in-china.md
```
Comment (huan) [inline .gitignore]: Why do we add this post to `.gitignore`?

**Review by huan — CHANGES_REQUESTED** [review]
~~outdated~~

File: docusaurus/tsconfig.json [hunk 2]
```diff
@@ -10,5 +10,9 @@
   "include": [
     "src/",
     "sidebar.ts",
+  ],
+  "lib": [
+    "es2016",
```
Comment (huan) [inline docusaurus/tsconfig.json]: How about using `es2020`? :)

**Review by huan — DISMISSED** [review]
LGTM

Thanks for the update!

**Review by shraddhavp — DISMISSED** [review]

**Review by lijiarui — DISMISSED** [review]
LGTM

**Comment by iamrajiv** [comment]
@huan Tests are failing it's saying `HTMLElement is not defined` I tried a lot to fix it but was not able to get it can you help, please. Btw I think it's not relevant we can ignore the tests.

**Comment by huan** [comment]
We have to fix this before merge because if this PR can not pass the CI, then it will block the whole repo to be published after it has been merged.

**Review by huan — APPROVED** [review]
LGTM

**Comment by huan** [comment]
Thank you very much for the improvements!

**Pull request merged**